Chapters/Expressions/Expressions.pillar

```diff
@@ -1047,13 +1052,13 @@ ENegation >> evaluateWith: anObject
 
 
 !!! Conclusion
-This little exercise was full of learning potential. Here is a little summary of what we explained and we hope you understood. 
-
-- A message specifies an intent while a method is a named list of execution. We often have one message and a list of methods with the same name. 
-- Sending a message is finding the method corresponding to the message selector: this selection is based on the class of the object receiving the message. When we look for a method we start in the class of the receiver and go up the inheritance link. 
-- Tests are a really nice way to specify what we want to achieve and then to verify after each change that we did not break something. Tests do not prevent bugs but they help us building confidence in the changes we do by identifying fast errors. 
-- Refactorings are more than simple code transformations. Usually refactorings pay attention that their application does not change the behavior of program. As we saw refactorings are powerful operations that really help doing complex operations in a few actions. 
-- We saw that the introduction of a new superclass and moving instance variables or method to a superclass does not change the structure or behavior of the subclasses. This is because (1) for the state, the structure of an instance is based on the state of its class and all its superclasses, (2) the lookup starts in the class of the receiver and look in superclasses. 
-- Each time we send a message, we create a potential place (a hook) for subclasses to get their code definition used in place of the superclass's one.
+This little exercise was full of learning potential. Here is a little summary: 
+
+- A ''message'' specifies an intent while a ''method'' is a named list of Pharo expressions. We often have one message and many methods with the same name. 
```

Comment:
Pharo instructions is better than expressions because some statements are not expressions but instructions.

Reply:
Perfect!

Reply:
I think I might have made a similar error elsewhere in the text; I'll update that too.